doc/src/build/install.md

```diff
@@ -4,69 +4,67 @@ title: Install Sui to Build
 
-## Choose the documentation branch
+## Documentation in the Sui repository
 
-This documentation is built from the same branches, `main` and `devnet`. The `main` branch includes the latest additions and updates to the documentation. You can view the content to learn about upcoming updates to the documentation, but the information may not be accurate or up-to-date for the features and functionality available in the `devnet` branch. In most cases, you should view the `devnet` version of the documentation.
+The `main` and `devnet` branches of the Sui repository contain the relevant documentation for each branch. A version toggle on the documentation site enables you to switch between `main` branch content (labeled **Latest build**) and `devnet` branch content (labeled **Devnet**). Make sure the toggle is set to **Devnet** to learn how to install, configure, and build on Sui. The content in **Latest build** is useful to learn about potential updates to Sui, but is not guaranteed to be accurate for the features and functionality available in the `devnet` branch.  
 
-To change branches, choose **Latest build** to view the documentation generated from the `main` branch of the repository. You should not use the **Latest build** version to learn how to install, configure, or build on Sui, as the information may change before the content is merged to **Devnet**. 
+## Supported operating systems
 
-## Supported Operating Systems
-
-Sui supports the following operating systems.
+Sui supports the following operating systems, beginning with the versions indicated.
 
 * Linux - Ubuntu version 18.04 (Bionic Beaver)
 * macOS - macOS Monterey
 * Microsoft Windows - Windows 11
 
 ## Prerequisites
 
-Install the prerequisites and tools you need to work with Sui. 
+Install the prerequisites and tools you need to work with Sui. Click the 
 
 | Package/OS | Linux  | macOS | Windows 11 |
 | --- | :---: | :---: | :---: |
-| Curl | X | X | X |
-| Rust | X | X | X |
-| Git CLI | X | X | X |
-| CMake | X | X | X |
-| libssl-dev | X | | |
-| libclang-dev | X | | |
-| Brew | | X | |
-| C++ build tools | | | X |
-| LLVM Compiler | | | X |
+| cURL | [X](#curl) | [X](#curl-1) | [X](#curl-2) |
+| Rust and Cargo | [X](#rust-and-cargo) | [X](#rust-and-cargo) | [X](#rust-and-cargo) |
+| Git CLI | [X](#git-cli) | [X](#git-cli-1) | [X](#git-cli-2) |
+| CMake | [X](#cmake) | [X](#cmake-1) | [X](#cmake-2) |
+| libssl-dev | [X](#libssl-dev) | | |
+| libclang-dev | [X](#libclang-dev) | | |
+| Brew | | [X](#brew) | |
+| C++ build tools | | | [X](#additional-tools-for-windows) |
+| LLVM Compiler | | | [X](#additional-tools-for-windows) |
 
 
 ### Rust and Cargo
 
-Sui requires Rust and Cargo on all supported operating systems. 
+Sui requires Rust and Cargo on all supported operating systems. Some operating systems require cURL to download Rust and Cargo, so check the relevant prerequisite section to install cURL first, if necessary.
```

Comment:
Can we just say which of the 3 requires cURL to install it? If you use macOS, cURL comes with the OS, for Windows you need to install C++ build tools before you can install Rust